docs/_data/img.yml

```diff
@@ -0,0 +1,3 @@
+nav:
+  arrow: /assets/img/expand_arrow.svg
+  menu: /assets/img/menu_icon.svg
```

Comment:
The ⛔ above is because GitHub expects that each file ends with one and only one carriage return.